dissect/target/helpers/record.py

```diff
@@ -142,3 +142,41 @@ def DynamicDescriptor(types):  # noqa
     "empty",
     [],
 )
+
+COMMON_INTERFACE_ELEMENTS = [
+    ("string", "name"),
+    ("string", "type"),
+    ("boolean", "enabled"),
+    ("string", "mac"),
+    ("net.ipaddress", "ip"),
+    ("net.ipaddress", "gateway"),
+    ("string", "source"),
+]
+
+
+UnixInterfaceRecord = TargetRecordDescriptor(
+    "unix/interface",
```

Comment:
```suggestion
    "unix/network/interface",
```
Maybe? And for the others.